### ISSUE TITLE
i3xrocks-net-traffic: Floods journalctl / logs every 5 seconds (needs a debug flag?)

### ISSUE THREAD

**Issue by eduncan911** [regolith-linux/regolith-desktop#538]
- [X] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [X] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [ ] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)
^- Slack is not allowed at my workplace.

**Describe the bug**
When you have the `i3xrocks-net-traffic` package installed, there seems to be a script that runs every 5 seconds - which echos the script's contents to `journalctl` on every run.

**To Reproduce**

1. Installed a fresh Pop_OS! today
2. Installed Regolith-desktop today
3. `$ journalctl -f`

And watch it flood...  `i3xrocks-net-traffic` is included by default. 

**Expected behavior**
Nothing is logged.  Only if you are running under some DEBUG flag or build should it echo its contents.

**Screenshots**
```
Jan 20 16:47:13 pop-os regolith.desktop[8159]: ++ cat /sys/class/net/wlp0s20f3/statistics/rx_bytes
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + RX2=2131035
Jan 20 16:47:13 pop-os regolith.desktop[8160]: ++ cat /sys/class/net/wlp0s20f3/statistics/tx_bytes
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + TX2=65179
Jan 20 16:47:13 pop-os regolith.desktop[8162]: ++ echo '(2131035-2130755)/5'
Jan 20 16:47:13 pop-os regolith.desktop[8163]: ++ bc
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + RX_DIFF=56
Jan 20 16:47:13 pop-os regolith.desktop[8165]: ++ echo '(65179-65179)/5'
Jan 20 16:47:13 pop-os regolith.desktop[8166]: ++ bc
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + TX_DIFF=0
Jan 20 16:47:13 pop-os regolith.desktop[8168]: ++ echo 56
Jan 20 16:47:13 pop-os regolith.desktop[8169]: ++ numfmt --to iec --format %f --padding 5
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + RX='   56'
Jan 20 16:47:13 pop-os regolith.desktop[8171]: ++ echo 0
Jan 20 16:47:13 pop-os regolith.desktop[8172]: ++ numfmt --to iec --format %f --padding 5
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + TX='    0'
Jan 20 16:47:13 pop-os regolith.desktop[8174]: ++ echo '(56+0)'
Jan 20 16:47:13 pop-os regolith.desktop[8175]: ++ bc
Jan 20 16:47:13 pop-os regolith.desktop[8176]: ++ numfmt --to iec --format %f --padding 5
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + AX='   56'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + [[    56 =~ [BKMG]$ ]]
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + RX='  56B'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + [[     0 =~ [BKMG]$ ]]
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + TX='   0B'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + [[    56 =~ [BKMG]$ ]]
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + AX='  56B'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + VALUE_COLOR='#bdae93'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + LABEL_COLOR='#a89984'
Jan 20 16:47:13 pop-os regolith.desktop[8177]: ++ xrescat i3xrocks.value.font 'Source Code Pro Medium 13'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + VALUE_FONT='JetBrains Mono Medium 13'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + '[' '' = up ']'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + '[' '' = down ']'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + '[' '' = dn ']'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + '[' '' = total ']'
Jan 20 16:47:13 pop-os regolith.desktop[8178]: ++ lspan 
Jan 20 16:47:13 pop-os regolith.desktop[8178]: ++ echo '<span color="#a89984"></span>'
Jan 20 16:47:13 pop-os regolith.desktop[8179]: ++ fspan '  56B'
Jan 20 16:47:13 pop-os regolith.desktop[8179]: ++ echo '<span font_desc="JetBrains Mono Medium 13" color="#bdae93">   56B</span>'
Jan 20 16:47:13 pop-os regolith.desktop[8180]: ++ lspan 
Jan 20 16:47:13 pop-os regolith.desktop[8180]: ++ echo '<span color="#a89984"></span>'
Jan 20 16:47:13 pop-os regolith.desktop[8181]: ++ fspan '   0B'
Jan 20 16:47:13 pop-os regolith.desktop[8181]: ++ echo '<span font_desc="JetBrains Mono Medium 13" color="#bdae93">    0B</span>'
Jan 20 16:47:13 pop-os regolith.desktop[8182]: ++ lspan 
Jan 20 16:47:13 pop-os regolith.desktop[8182]: ++ echo '<span color="#a89984"></span>'
Jan 20 16:47:13 pop-os regolith.desktop[8080]: + echo '<span color="#a89984"></span><span font_desc="JetBrains Mono Medium 13" color="#bdae93">   56B</span><span color="#a89984"></span><span font_desc="JetBrains Mono Medium 13" color="#bdae93">    0B</span><span color="#a89984"></span>'
Jan 20 16:47:13 pop-os regolith.desktop[8185]: ++ xrescat i3xrocks.label.up ' up'
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + UP=
Jan 20 16:47:13 pop-os regolith.desktop[8186]: ++ xrescat i3xrocks.label.dn ' dn'
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + DN=
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + DLY=5
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + ERROR_DLY=10
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + RT=
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + NUMFMT='numfmt --to iec --format %f --padding 5'
Jan 20 16:47:13 pop-os regolith.desktop[8188]: ++ ip route show default
Jan 20 16:47:13 pop-os regolith.desktop[8189]: ++ awk '!/(ppp|tun|tap)/ { print $5 }'
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + IF=wlp0s20f3
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + IF_PATH=/sys/class/net/wlp0s20f3
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + [[ wlp0s20f3 == '' ]]
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + [[ ! -d /sys/class/net/wlp0s20f3 ]]
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + [[ ! -f /sys/class/net/wlp0s20f3/statistics/rx_bytes ]]
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + [[ ! -f /sys/class/net/wlp0s20f3/statistics/tx_bytes ]]
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + '[' -d /sys/class/net/wlp0s20f3/wireless ']'
Jan 20 16:47:13 pop-os regolith.desktop[8190]: ++ xrescat i3xrocks.label.wifi 
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + NIC=
Jan 20 16:47:13 pop-os regolith.desktop[8191]: ++ cat /sys/class/net/wlp0s20f3/statistics/rx_bytes
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + RX1=2131035
Jan 20 16:47:13 pop-os regolith.desktop[8192]: ++ cat /sys/class/net/wlp0s20f3/statistics/tx_bytes
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + TX1=65179
Jan 20 16:47:13 pop-os regolith.desktop[8184]: + sleep 5
```

Notice the `sleep 5` as the last line?

**Configuration file(s)**
Nothing really.  But here's all I've done total:

```
$ cat ~/.config/regolith/Xresources 
Xft.dpi: 192
Xcursor.size: 48
i3-wm.bar.position: top

$ sudo apt search regolith | grep installed

i3xrocks/groovy,now 1.3.4-1 amd64 [installed,automatic]
regolith-compositor-picom-glx/groovy,now 1.1.3-1 amd64 [installed,automatic]
regolith-default-settings/groovy,now 1.0.5-1groovy amd64 [installed,automatic]
regolith-desktop-standard/groovy,now 2.106-1groovy amd64 [installed]
regolith-ftue/groovy,now 1.1.0-1 amd64 [installed,automatic]
regolith-gnome-flashback/groovy,now 2.6.2-1 amd64 [installed,automatic]
regolith-i3-gaps-config/groovy,now 2.8.4-1 amd64 [installed,automatic]
regolith-i3xrocks-config/groovy,now 3.5.6-1 amd64 [installed,automatic]
regolith-look-ayu/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-ayu-dark/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-ayu-mirage/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-cahuella/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-dracula/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-gruvbox/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-lascaille/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-nord/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-pop-os/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-solarized-dark/groovy,now 2.6.14-1 amd64 [installed]
regolith-look-ubuntu/groovy,now 2.6.14-1 amd64 [installed]
regolith-rofi-config/groovy,now 1.3.1-1 amd64 [installed,automatic]
regolith-rofication/groovy,now 1.3.1-1 amd64 [installed,automatic]
regolith-styles/groovy,now 2.6.14-1 amd64 [installed,automatic]
regolith-unclutter-xfixes/groovy,now 1.5-2 amd64 [installed,automatic]
remontoire/groovy,now 1.4.0-1 amd64 [installed,automatic]

$ sudo apt search i3xrocks | grep installed

i3xrocks/groovy,now 1.3.4-1 amd64 [installed,automatic]
i3xrocks-battery/groovy,now 3.5.6-1 amd64 [installed]
i3xrocks-cpu-usage/groovy,now 3.5.6-1 amd64 [installed,automatic]
i3xrocks-key-indicator/groovy,now 3.5.6-1 amd64 [installed]
i3xrocks-net-traffic/groovy 3.5.6-1 amd64 [installed,automatic]
i3xrocks-rofication/groovy,now 3.5.6-1 amd64 [installed,automatic]
i3xrocks-temp/groovy,now 3.5.6-1 amd64 [installed]
i3xrocks-time/groovy,now 3.5.6-1 amd64 [installed,automatic]
i3xrocks-todo/groovy,now 3.2.6-1ubuntu2 amd64 [installed]
i3xrocks-volume/groovy,now 3.5.6-1 amd64 [installed]
regolith-i3xrocks-config/groovy,now 3.5.6-1 amd64 [installed,automatic]
```

**Installation Details**
 - Regolith Install Type: PPA
 - Regolith Version: 2.106-1groovy (desktop)
 - PPA url: [default: ppa:regolith-linux/release]
 - Host OS (for PPA): Pop_OS 20.10

**Additional context**
I recommend hiding these entries under a debug flag.

**Comment by kgilmer** [comment]
Hi @eduncan911 ,

Yes a debug line unfortunately made it into the release in the net-traffic script.  I've pasted a notice I made yesterday in slack to give you some short term options at resolution until the next release is available:

```
There is a non-functional bug in Regolith 1.5.2 in the net-traffic status indicator.  I forgot to remove a debugging line before release.  This means 10 - 20 lines of debug output are added to /var/log/syslog every 10 seconds.  A fix was published to unstable PPA about 8 hours ago.  The fix is currently propagating to stable PPA.  If you're on the release PPA and don't want to wait for the next release (we will issue another bug fix release, 1.5.3), you can simply uninstall i3xrocks-net-traffic or remove line 5 in /usr/share/i3xrocks/net-traffic (https://github.com/regolith-linux/regolith-i3xrocks-config/blob/a9b47d4d43c4561405b3736187f4f9f4a5e403e8/scripts/net-traffic#L5)
```

**Comment by eduncan911** [comment]
Most excellent answer, thank you!

>  uninstall i3xrocks-net-traffic

Yep, that's what I did.

**Comment by sp33dsk8** [comment]
Here is a hotfix if you'd like to keep the net-traffic but not the debug spammy spam
This sed will replace line 5 "set" with "#set"

```
$ sudo sed -i "5 s/set/#set/" /usr/share/i3xrocks/net-traffic
$ i3-msg restart
```

**Comment by albertlincoln** [comment]
I changed line 618 of /etc/regolith/i3/config to:

`status_command 2>/dev/null $i3-wm.bar.status_command`

**Comment by kgilmer** [comment]
This has been resolved in Regolith 1.5.3

**Comment by eduncan911** [comment]
I haven't updated yet but when I do I'll test it.  If it still occurs, I'll
come back.

Thanks again!
